Add CNAME trackers #76163

AdamWr opened the pull request:
https://github.com/AdguardTeam/AdguardFilters/issues/76163

TODO:
- [x] Exclude these trackers which may be unsafe

AdamWr committed:
Add CNAME trackers #76163

Yuki2718 commented:
It seems most of pardot cname tracker are used as click-through so can't entirely be blocked. These are what I confirmed should not be blocked at least as 1st party:
```
! biz-sol.sony.jp
! http://www.hoteresonline.com/articles/6971

! a.nissyoku.co.jp
! https://a.nissyoku.co.jp/l/737513/2019-12-05/pkf2

! biz-info.fielding.co.jp
! https://biz-info.fielding.co.jp/nec-fielding_contact

! biz.mkt.global.dnp.co.jp
! https://biz.mkt.global.dnp.co.jp/l/819623/2020-06-26/2hbbv

! biz.optage.co.jp
! https://biz.optage.co.jp/5g/news/20201012/
```

ameshkov commented:
I am also not entirely sure about Act-On, I don't see it blocked at all in AG Tracking Protection at the moment (`actonsoftware.com` and `actonservice.com`). Maybe there are other trackers like that as well?

@AdamWr let's start slow and only block those that are 100% blocked in AG Tracking Protection filter, and then we'll slowly add others when we're sure it's okay to block it.

AdamWr committed:
Removed some trackers

Removed actonservice.com, ghochv3eng.trafficmanager.net, tracking.bp01.net, go.pardot.com, tracedock.com 
Added a8.net
Added section for fixing incorrect blocking
